modules/core/src/main/scala/shop/Main.scala

```diff
@@ -21,7 +22,7 @@ object Main extends IOApp {
             clients <- HttpClients.make[IO](cfg.paymentConfig, res.client)
             programs <- Programs.make[IO](cfg.checkoutConfig, algebras, clients)
             api <- HttpApi.make[IO](algebras, programs, security)
-            _ <- BlazeServerBuilder[IO]
+            _ <- BlazeServerBuilder[IO](ExecutionContext.global: ExecutionContext)
```

Comment:
I think the type annotation isn't necessary, is it? It should compile with just `BlazeServerBuilder[IO](ExecutionContext.global)`